src/ServiceControl.Monitoring/Infrastructure/IKnowAboutEndpoints.cs

```diff
@@ -0,0 +1,45 @@
+﻿namespace ServiceControl.Monitoring.Infrastructure
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using Timings;
+
+    public interface IKnowAboutEndpoints
+    {
+        IDictionary<string, IEnumerable<EndpointInstanceId>> AllEndpointData();
+    }
+
+    public interface IProvideEndpointMonitoringData
```

Comment:
Yeah. I think that's the next logical step. Then adding a new one is literally adding an interface to a class.